### ISSUE TITLE
YAML Workflow Library not being included in PyPi package

### ISSUE THREAD

**Issue by shyamd** [hackingmaterials/atomate#132]
Installing atomate from pypi doesn't include the library folder with all the yaml definitions for basic workflows. This folder should probably be included in there. I'm not sure how to change this or I'd put in a pull request myself.

**Comment by matk86** [comment]
It should be added to manifest.in file

**Comment by shyuep** [comment]
Pushed a fix. @computron to do release.

**Comment by shyuep** [comment]
Also note that currently, atomate on pypi is a gigantic 55Mb download. I added pruning of tests and test_files to the MANIFEST.in. That brings it down to a sane 1Mb.

**Comment by shyuep** [comment]
Reopening. @computron should check and confirm all files and release new version before closing.

**Comment by matk86** [comment]
library is still missing

**Comment by computron** [comment]
Ok as of atomate 0.5.4 the library is there. I never understood why there is both MANIFEST.in as well as ``package_data`` arg in setup.py, but I think the latter needed to be updated with:

```
package_data={'atomate.vasp.workflows.base': ['library/*']}
```

@shyuep mentions that the PyPI download still includes test files, making the pip download 55MB. That is large but I would say it's a separate issue to open - closing this one

**Comment by computron** [comment]
Ok PyPI download size should also be fixed as of v0.5.5
